assets/component-menu-drawer.css

```diff
@@ -224,11 +224,20 @@ details[open].menu-opening > .menu-drawer__submenu {
   align-items: center;
   text-decoration: none;
   padding: 1.2rem;
-  margin-left: -1.2rem;
   font-size: 1.4rem;
   color: rgb(var(--color-foreground));
 }
 
+@media screen and (min-width: 750px) {
+  .menu-drawer__account {
+    display: none;
+  }
+
+  .menu-drawer__utility-links:has(ul:empty) {
```

Comment:
`:has` isn't supported very well so we haven't been using it so far in the theme. 
Maybe it's ok here since for older browser version or firefox it will just show a little grey box which isn't super problematic. 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/17-07-g69pv-33haa.png)
</details>

Reply:
I guess the alternative would be to do something like @LucasLacerdaUX did here where we check in Liquid if the social icons are set and if so omit the whole area: https://github.com/Shopify/dawn/pull/2226

Personally I am OK with a progressive enhancement here. It will work for almost all buyers and not look terribly broken for the rest.